subsys/bluetooth/host/hci_core.c

```diff
@@ -4081,6 +4081,9 @@ int bt_disable(void)
 	 */
 	atomic_clear_bit(bt_dev.flags, BT_DEV_ENABLE);
 
+	/* Set to initial value */
+	hci_cmd_pool.uninit_count = CONFIG_BT_BUF_CMD_TX_COUNT;
```

Comment:
Move this change to its own commit (still in this same PR), since it's not specific to the rpmsg driver.

Reply:
Actually, why is this change needed? net_buf pools should be self-managing as long as you're correctly allocating and freeing (unreffing) buffers.

Reply:
Or to put it another way `net_buf_poo.uninit_count` should be seen as an internal variable to the net_buf_pool implementation - you should never need to touch it from outside of the net_buf subsystem.

Reply:
@jhedberg I'll address all your comments today later. It's hard to say why I did that (I was working on it in March) as I am checking it now it looks like this structure is initialized statically. As I remember there was a wrong initial value after reenabling BT.

Reply:
@jhedberg One more time checked that and now it seems to work without touching hci_cmd_pool. This modification is removed.